tests/dummy/app/templates/overview.hbs

```diff
@@ -35,7 +37,7 @@
           <small>(only core, widget, mouse, resizable, and sortable modules required)</small>
         </li>
         <li><a target="_BLANK" href="https://github.com/brandonaaron/jquery-mousewheel">jquery.mousewheel</a></li>
-        <li><a target="_BLANK" href="https://github.com/LearnBoost/antiscroll">antiscroll</a></li>
+        <li><a target="_BLANK" href="https://github.com/azirbel/antiscroll">antiscroll</a></li>
```

Comment:
We do have custom modifications, and even though the project is transferred, PRs are being ignored. We can move this to an Addepar repository, though. I'll add that to my list and do it as a separate project.